pkg/cmd/cli/secrets/unlink_secret_from_object.go

```diff
@@ -0,0 +1,118 @@
+package secrets
+
+import (
+	"fmt"
+	"io"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	UnlinkSecretRecommendedName = "unlink"
+
+	// TODO: move to examples
+	unlinkSecretLong = `
+Unlink (detach) secrets from a service account
```

Comment:
I was expecting a bit more description.  I don't think the one line here tells me *why* I link or unlink